JuiceMaker/JuiceMaker/Controller/FruitQuantityViewController.swift

```diff
@@ -10,7 +10,85 @@ import UIKit
 final class FruitQuantityViewController: UIViewController {
     static let id = String(describing: FruitQuantityViewController.self)
     
+    @IBOutlet weak var kiwiLabel: UILabel!
+    @IBOutlet weak var mangoLabel: UILabel!
+    @IBOutlet weak var pineappleLabel: UILabel!
+    @IBOutlet weak var bananaLabel: UILabel!
+    @IBOutlet weak var strawberryLabel: UILabel!
+    
+    @IBOutlet weak var strawberryStepper: UIStepper!
+    @IBOutlet weak var bananaStepper: UIStepper!
+    @IBOutlet weak var pineappleStepper: UIStepper!
+    @IBOutlet weak var kiwiStepper: UIStepper!
+    @IBOutlet weak var mangoStepper: UIStepper!
+    
+    private let juiceMaker = JuiceMaker()
+    private let fruitStore = FruitStore.shared
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        hideBackButton()
+        updateStockToLabel()
+    }
+    
+    private func hideBackButton() {
+        navigationItem.hidesBackButton = true
+    }
+    
+    private func transformStringToInt(label: String?) -> Int {
+        guard let stringLabel = label, 
+                let number = Int(stringLabel) else { return 0 }
+        return number
+    }
+    
+    @IBAction func stepperPressed(_ sender: UIStepper) {
+        let value = Int(sender.value)
+        
+        switch sender {
+        case strawberryStepper:
+            strawberryLabel.text = String(value)
+        case bananaStepper:
+            bananaLabel.text = String(value)
+        case pineappleStepper:
+            pineappleLabel.text = String(value)
+        case kiwiStepper:
+            kiwiLabel.text = String(value)
+        case mangoStepper:
+            mangoLabel.text = String(value)
```

Comment:
```suggestion
for fruit in Fruit.allCases {
            guard sender == convertFruitsToUIComponents(from: fruit).stepper else { continue }
        }
```

이런식으로 switch 문을 타지 않더라도 guard 문으로 sender 가 어떤 stepper 인지 특정할 수 있습니다